installation/linux.md

```diff
@@ -64,7 +73,7 @@ Make sure to run uninstallation process as root.
 systemctl stop edgedelta.service
 systemctl disable edgedelta.service
 rm /etc/systemd/system/edgedelta.service
-rm /usr/lib/systemd/system/edgedelta.service 
+rm /usr/lib/systemd/system/edgedelta.service
```

Comment:
I guess some of these commands will need `sudo` as well

Reply:
I removed this section and added uninstall script with sudo.